src/test/test-udev.c

```diff
@@ -55,7 +55,7 @@ static int fake_filesystems(void) {
 }
 
 static int run(int argc, char *argv[]) {
-        _cleanup_(udev_rules_unrefp) struct udev_rules *rules = NULL;
+        _cleanup_(udev_rules_freep) struct udev_rules *rules = NULL;
```

Comment:
(btw, in general i think it's ok to call destructors _unref() even if they have no ref counter. Whether they do is an implementation detail ultimately, and there's no real reason to make this super-visible in the naming of the destructor calls...

Sometimes in particular in public APIs it might be worse preparing for the future, i.e. name things _unref() even if it currently doesn#t do ref counting, so that one day it might gain support for it without breaking API...)

Reply:
I see. Thanks.

Reply:
... but, this is an internal API. I think it's better to name the function as precisely as possible, just to make reading easier. If somebody ever decides to add ref counting, doing a `sed -i` over the codebase is trivial.